### PR TITLE
Added function to ingest data from csv file

### LINKED ISSUE #30: Write code to ingest given data from csv
We have obtained location data from the city of seattle, and it was given in a csv format. We need to write code to actually ingest the data into our DynamoDB backend.

### PR DESCRIPTION
**Pull request by kumararoosh**
Resolves #30

### CODE REVIEW

File: FACBackEnd/src/ingestData.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+const csv = require('csv-parser')
+const fs = require('fs')
+const putItem = require('./putItem')
+
+function readFromCSV (relativePath) {
```
Comment (ubelhj) [inline FACBackEnd/src/ingestData.js:14]: Needs comments to explain usage

File: FACBackEnd/src/ingestData.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  fs.createReadStream('./sampledata/seattlecanlocations.csv')
```
Comment (ubelhj) [inline FACBackEnd/src/ingestData.js]: Hardcoded file path is not good. Should likely be relativePath. Also unclear what the format of the .csv needs to be

File: FACBackEnd/src/ingestData.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    .pipe(csv())
+    .on('data', (row) => {
+      // obj = JSON.parse(row)
+      putItem.putItem('' + row.LATITUDE, '' + row.LONGITUDE, Boolean(row.isGarbage), Boolean(row.isCompost), Boolean(row.isRecycling))
```
Comment (ubelhj) [inline FACBackEnd/src/ingestData.js:18]: Needs to clearly tell the user what the .csv should look like

**Review by ubelhj — CHANGES_REQUESTED** [review]
Relies heavily on pre-conditions that aren't made clear by the function

File: FACBackEnd/src/ingestData.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+          if (error) console.log(error)
+          else console.log(result)
+        })
+        // console.log(Boolean(row.isGarbage))
+    })
+    .on('end', () => {
+      console.log('finished')
+    })
+}
+
```
Comment (jimmyguo8) [inline FACBackEnd/src/ingestData.js:30]: Can add comments on expected output on success and failure

File: FACBackEnd/src/ingestData.js (hunk 2 above)
Comment (jimmyguo8) [reply FACBackEnd/src/ingestData.js]: Add comment on the form of what the csv should be in

**Review by jimmyguo8 — COMMENTED** [review]
Just needs additional comments that better explain the function

**Pull request merged**